### ISSUE TITLE
Generated SQL statement contains wrong bindings

### ISSUE THREAD

**Issue by laacz** [glhd/laravel-dumper#6]
**Describe the bug**
When dumping Query Builder, it generates incorrect SQL statement.

**What version does this affect?**
 - Laravel Version: 9.0.0
 - Package Version: 1.0.0

**To Reproduce**
```php
dd(User::where('name', 'test')->whereBetween('id', [1, 2]));
```

It generates following SQL: `select * from "users" where "name" = 'test' and "id" between 'test' and 'test'`

**Expected behavior**

SQL: `select * from "users" where "name" = 'test' and "id" between 1 and 2`

**Additional context**

That lost me a few hours, trying to hunt down the bug in my app, then in Eloquent itself, then checking if it is affected by me being on PHP 8.1, etc.

**Comment by inxilpro** [comment]
Oof. Sorry about how much time that cost you! This is fixed in the [1.0.1 release](https://github.com/glhd/laravel-dumper/releases/tag/1.0.1).

**Comment by laacz** [comment]
Thanks. That's alright and is an unavoidable part of a developer's life :)
